compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -896,6 +898,35 @@ void QuantizeWithMinMaxPass::set_output_type(loco::Graph *g) const
   }
 }
 
+LayerInfoMap QuantizeWithMinMaxPass::create_layer_info_map(loco::Graph *g) const
+{
+  auto info_by_name = LayerInfoMap();
+
+  for (auto &&info : _ctx->layers_info)
+  {
+    auto name = info.name;
+    bool found = false;
+    for (auto node : loco::active_nodes(loco::output_nodes(g)))
+    {
+      auto cnode = loco::must_cast<luci::CircleNode *>(node);
+      if (cnode->name() == name)
+      {
+        info_by_name[name] = &info;
```

Comment:
circle graph assumes `name` should be unique. (there is validation but duplicate can splip in)
Q) is there any assumptions in `create_layer_info_map` about the name as this returns `map` ?

Reply:
Hmm.. There is no assumption in `_ctx->layers_info`. I should add a logic to handle duplicate names. Thanks!